features/extract_command.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Extracting theme contents to source
```

Comment:
I'm not seeing a test for any of the sad paths that could be here. Could you add a few please?